core/blockly.js

```diff
@@ -198,6 +198,17 @@ exports.VERSION = 'uncompiled';
 // Blockly.mainWorkspace, Blockly.prompt and Blockly.selected for backwards
 // compatibility.
 Object.defineProperties(exports, {
+  /**
+   * Wrapper to window.alert() that app developers may override to
+   * provide alternatives to the modal browser window.
+   * @name Blockly.alert
+   * @function
```

Comment:
This strikes me as odd - I feel like it should be `@type {!function(string, function()=)}` which would more closely match the other properties, but I can't really find anything concrete about how to do this in general so I'm not sure. This is just my intuition. I think because this way I read it as the set and get functions take these params, which isn't quite right. But again, not sure of that.

Any thoughts?

Reply:
I agree, especially as there is not actually a `function` expression to match the parameter names against.  I think it ended up as shown because I'd been copying verbatim from the previous declarations (i.e., back when these were mutable data properties).  Fixed.

(FYI: This declaration documents the type of the 'virtual' accessor property, rather than the the types of the accessors themselves.  The `@name` means that [the compiler should be ignoring the surrounding code](https://jsdoc.app/tags-name.html), but apparently Closure Compiler has a bug which forces me to put this JSDoc in 
 a place where a JSDoc are legal and then do a `@suppress {checkTypes}` because the thing that the JSDoc is actually documenting (`Blockly.alert`) is not the thing the compiler mistakenly thinks it is attached to (the `alert` property of the anonymous object passed to `Object.defineProperties`.  Ugh.)